src/ceph-daemon

```diff
@@ -31,8 +31,15 @@ You can invoke ceph-daemon in two ways:
 """
 
 import argparse
-import configparser
+try:
+    from configparser import ConfigParser   # py3
```

Comment:
these imports should be swapped to try Python2 first. It was painful to debug, but it caused issues in ceph-medic (see https://github.com/ceph/ceph-medic/commit/f499a89b15b72a5bb842e1ebbc7cafc4daf58c1c )